src/index.html

```diff
@@ -4,9 +4,373 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/reset.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__items">
+          <li class="nav__item"><a class="nav__link is-active" href="#">Apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Smartphones</a></li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a class="nav__link" href="#">Gadgets</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Tablets</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Photo</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main main--content">
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="product image"
+        >
+        <div class="card__description">
```

Comment:
[CODE STYLE] - Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements

Reply:
@ivaZaiets check this point in checklist also)